src/components/Header/Header.css

```diff
@@ -1,56 +1,328 @@
 }
 
 .header__main {
-    position: relative;
-    max-width: 330px;
-    margin: 0 auto;
-    display: flex;
-    flex-direction: column;
-    margin-top: 160px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  margin-top: 90px;
 }
 
 .header__logo {
-    position: absolute;
-    top: 0;
-    left: 37px;
+  width: 23px;
+  height: 28px;
+  position: absolute;
+  top: 0;
+  left: 37px;
 }
 
 .header__title {
-    color: #fff;
-    font-size: 34px;
-    font-style: normal;
-    font-weight: 700;
-    line-height: 43px;
-    letter-spacing: 0em;
-    text-align: center;
-    margin: 0;
-    padding: 0;
-    padding-top: 28px;
+  color: #fff;
+  font-size: 34px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 43px;
+  letter-spacing: 0em;
+  text-align: center;
+  margin: 0;
+  padding: 0;
+  padding-top: 28px;
 }
 
 .header__subtitle {
-    color: #a0a0a0;
-    font-size: 14px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: 20px;
-    text-align: center;
-    margin-top: 16px;
-    margin-bottom: 90px;
-    padding: 0;
+  max-width: 320px;
+  color: #a0a0a0;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  text-align: center;
+  margin: 16px auto 92px;
+  padding: 0;
 }
 
 .header__buttons {
-    display: flex;
-    gap: 10px;
+  display: flex;
+  gap: 10px;
+}
+
+@keyframes straight-Y {
+  0%,
+  100% {
+    transform: translateY(0);
+  }
+  50% {
+    transform: translateY(-145px);
+  }
+}
+
+@keyframes left-curve-X {
+  0%, 100% {
+    bottom: 30px;
+    left: 156px;
+  }
+  40%, 60% {
+    transform: rotate(0);
+    bottom: 30px;
+    left: 10px;
+  }
+  50% {
+    transform: rotate(-90deg);
+    bottom: -2px;
+    left: -10px;
+  }
+}
+
+@keyframes left-curve-Y {
+  0%, 100% {
+    bottom: 0;
+    left: -14px;
+  }
+  40%, 60% {
+    transform: rotate(0);
+    bottom: 140px;
+    left: -14px;
+  }
+  50% {
+    transform: rotate(90deg);
+    bottom: 166px;
+    left: 14px;
+  }
+}
+
+@keyframes right-curve-X {
+  0%, 100% {
+    bottom: 78px;
+    left: 130px;
+  }
+  20%, 80% {
+    transform: rotate(0);
+    bottom: 78px;
+    left: 15px;
+  }
+  30%, 70% {
+    transform: rotate(-90deg);
+    bottom: 55px;
+    left: -8px;
+  }
+  50% {
+    bottom: 0;
+    left: -8px;
+  }
+}
+
+@keyframes right-curve-Y {
+  0%, 100% {
+    bottom: 0;
+    left: -11px;
+  }
+  20%, 80% {
+    transform: rotate(0);
+    bottom: 140px;
+    left: -11px;
+  }
+  25%, 75% {
+    transform: rotate(90deg);
+    bottom: 166px;
+    left: 14px;
+  }
+  50% {
+    bottom: 166px;
+    left: 125px;
+  }
+}
+
+.header__left-column {
+  position: relative;
+  display: flex;
+}
+
+.header__left-column :first-child {
+  position: absolute;
+  width: 4px;
+  height: 184px;
+  background-color: #fff;
+  bottom: -10px;
+  right: 100px;
+}
+
+.header__left-column :first-child:before {
+  content: '';
+  width: 40px;
+  height: 40px;
+  position: absolute;
```

Comment:
I try to avoid it when possible, but I used absolute positioning a _lot_ for the HeaderNav. The problem of trying to get pixel perfect alignment without it can be really tricky in some cases, and these weird blocks and curves are a case in point. Or anything that needs to be removed from the document flow, like the form or the HeaderNav.

Reply:
awesome, easier for me then as well 😂 i definitely used it a bit in carousel pr but had planned to go back to fix it—will likely just leave as is!

Reply:
I don't really know how to avoid using absolute positioning here, except for using percents instead of pixels. I will try this.